.github/workflows/main.yml

```diff
@@ -30,13 +30,13 @@ jobs:
     - name: Build
       run: dotnet build MonoDevelop.Xml.sln -c ${{ matrix.config }} --no-restore
 
-    - name: Test (Windows)
-      if: startsWith (matrix.os, 'windows')
-      run: dotnet test -c ${{ matrix.config }} --no-build
+    # - name: Test (Windows)
```

Comment:
Tests won't work unless minieditor gets updated.

Reply:
we don’t care at this point